web_src/js/features/issue-content-history.js

```diff
@@ -0,0 +1,135 @@
+import {svg} from '../svg.js';
```

Comment:
How about create a `issue` folder and make the js file as `features/issue/content-history.js`?

Reply:
Maybe it should be done when we are refactoring issue related JavaScript modules. I can feel we may have new thoughts or plans then.